jdaviz/configs/default/plugins/subset_plugin/subset_plugin.py

```diff
@@ -226,6 +226,104 @@ def vue_update_subset(self, *args):
             self.hub.broadcast(SnackbarMessage(
                 f"Failed to update Subset: {repr(err)}", color='error', sender=self))
 
+    def vue_recenter_subset(self, *args):
+        # Composite region cannot be edited. This only works for Imviz.
+        if not self.is_editable or self.config != 'imviz':  # no-op
+            return
```

Comment:
Would there be any downside to this raising an error instead?  The button shouldn't show in these cases, right?  But when we eventually move this to a public API, we'll probably want the error instead of the silent return?

Reply:
Hmm... Let's say this button is accidentally exposed under some condition. User clicks it for the wrong subset. Either way they are going to complain but which is better?

a. User sees long traceback.
b. Editable fields not updated. No traceback.